src/Components/Unsubscribe/Unsubscribe.js

```diff
@@ -96,11 +101,20 @@ class Unsubscribe extends Component {
             {this.state.error ? <Error error={this.state.error} /> :
               <form className="unsubscribe-form-container">
                 <div className="unsubscribe-form-header-container">
-                  <h2 className="unsubscribe-form-header">Unsubscribe from Election Reminders</h2>
+                  <h2 className="unsubscribe-form-header">
+                    <FormattedMessage
+                      id="unsubscribe.formHeader"
+                      defaultMessage="Unsubscribe from Election Reminders" />
+                  </h2>
                 </div>
                 <div className="unsubscribe-form-content-container">
                   <div className="unsubscribe-label-input-container unsubscribe-email-label">
-                    <label className="unsubscribe-label" htmlFor="email">Email Address<em>*</em></label>
+                    <label className="unsubscribe-label" htmlFor="email">
+                      <FormattedMessage
+                        id="unsubscribe.emailLabel"
+                        defaultMessage="Email Address" />
+                        <em>*</em>
+                    </label>
```

Comment:
Shoooot, you could even pull all of the labels out and put them into an external file too!